README.md

```diff
@@ -258,11 +258,17 @@ GitHub][phpenv-url]. It's clean, modular,
 and easy to understand (thanks to the rbenv project), even if you're not a
 shell hacker.
 
+Tests are executed using [Bats](https://github.com/bats-core/bats-code):
```

Comment:
I think this should be bats-core/bats-core no?

```suggestion
Tests are executed using [Bats](https://github.com/bats-core/bats-core):
```

Reply:
good call, will update